loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -506,7 +506,9 @@ export class GLTFLoader implements IGLTFLoader {
             for (let index of scene.nodes) {
                 const node = ArrayItem.Get(`${context}/nodes/${index}`, this._gltf.nodes, index);
                 promises.push(this.loadNodeAsync(`/nodes/${node.index}`, node, (babylonMesh) => {
-                    babylonMesh.parent = this._rootBabylonMesh;
+                    if (this._rootBabylonMesh) {
+                        babylonMesh.parent = this._rootBabylonMesh;
+                    }
```

Comment:
Is this necesary? If there is no rootBabylonMesh, then assigning the parent to undefined is fine right? Maybe make _rootBabylonMesh a Nullable<Mesh>?

Reply:
Changed by making it null by default as suggested.